freetextresponse/freetextresponse.py

```diff
@@ -446,18 +474,21 @@ def submit(self, data, suffix=''):
                 )
             )
         self.student_answer = data['student_answer']
-        if self.max_attempts == 0:
-            self.count_attempts = 1
-        else:
-            self.count_attempts += 1
-        self._compute_score()
+        word_count_message = self._get_word_count_message(1)
```

Comment:
This has to do with ignoring the fact the count_attempts might be zero. However, I am going to find a cleaner way of doing this when I commit the changes from this review